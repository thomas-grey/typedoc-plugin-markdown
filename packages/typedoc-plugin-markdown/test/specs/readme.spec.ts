import { expectFileToEqual } from '@devtools/testing';

describe(`Readmes`, () => {
  test(`should get standard readme `, () => {
    expectFileToEqual('reflections', 'members', 'README.md', 1);
  });

  test(`should get merged readme for members`, () => {
    expectFileToEqual('readme', 'members', ['index.md']);
  });

  test(`should get merged readme for modules`, () => {
    expectFileToEqual('readme', 'modules', ['index.md']);
  });
});
