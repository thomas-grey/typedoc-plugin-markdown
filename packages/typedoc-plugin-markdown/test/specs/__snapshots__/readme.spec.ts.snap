// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Readmes should get merged readme for members: (Output File Strategy "members") (Option Group "1") 1`] = `
"**typedoc-stubs v0.0**

***

# MY README

> Some block quote

Some text.

## Readme heading

Some text.

Module comments for documentation.

## Functions

- [someFunction](functions/someFunction.md)
"
`;

exports[`Readmes should get merged readme for modules: (Output File Strategy "modules") (Option Group "1") 1`] = `
"**typedoc-stubs v0.0**

***

# MY README

> Some block quote

Some text.

## Readme heading

Some text.

Module comments for documentation.

## Functions

### someFunction()

> **someFunction**(): \`void\`

#### Returns

\`void\`
"
`;

exports[`Readmes should get standard readme : (Output File Strategy "members") (Option Group "1") 1`] = `
"# MY README

> Some block quote

Some text.

## Readme heading

Some text.
"
`;
